mozfun/event_analysis/README.md

```diff
@@ -0,0 +1,19 @@
+# Event Analysis
+
+These functions are specific for use with the `events_daily` and `event_types` tables.
+By themselves, these two tables are nearly impossible to use since the event history
+is compressed; however, these stored procedures should make the data accessible.
+
+The `events_daily` table is created as a result of two steps:
+1. Map each event to a single UTF8 char which will represent it
+2. Group each client-day and store a string that records, using the
+    compressed format, that clients' event history for that day.
+    The characters are ordered by the timestamp which they appeared
+    that day.
+
+We primarily enable access to this data by creating views which do the heavy lifting.
```

Comment:
This slipped through before, but we suggest not using "we" in documentation:

https://docs.telemetry.mozilla.org/contributing/style_guide.html#writing-style

```suggestion
The best way to access this data is to create a view to do the heavy lifting.
```